# 2の課題

学生番号：

氏　　名：

#### ■1.  身長178.5cm、体重69.4kgwの人のBMIを求めよ:

In [1]:
69.4 / (178.5/100)**2

21.781261524217534

> $\rm{BMI}=\frac{体重}{身長^2}$　ただし，体重の単位は $\rm{kgf}$，身長の単位は $\rm{m}。$

#### ■2．131072ミリ秒について、分と秒で求めたい。次の手順で求めよ。

□2-1．131072ミリ秒の秒を求めよ：

In [2]:
131072 / 1000

131.072

> ヒント：ミリ( m )は$\frac{1}{1000}$のことだ。

□2-2．131072ミリ秒の分を求めよ：

In [3]:
131072 / 1000 // 60

2.0

□2-3. 131072ミリ秒を何分何秒にしたときの秒を求めよ：

In [4]:
(131072 / 1000) % 60

11.072000000000003

□2-4．結局131072ミリ秒は何分何秒か？下のマークダウンセルに答えよ：

2分11.072秒

#### ■3. 直円錐について以下の計算を行いなさい。
直円錐の底面の半径を$r$，高さを$h$とする。そのとき，
$$
体積=\frac{1}{3}\times h\times \pi r^2
$$
である。上の公式を用いて，底面半径3, 高さ5の直円錐の体積を求めよう。

□3-1. `math`モジュールをインポートせよ：

In [1]:
import math

□3-2. 直円錐の体積を求めよ：

In [2]:
(1/3) * 5 * math.pi * 3**2

47.12388980384689